contracts/staking/.cargo/config

```diff
@@ -0,0 +1,5 @@
+[alias]
```

Comment:
The .cargo-ok above should be deleted and `.gitignore`d. It is an artifact from cargo-generate.